app/src/forms/minesoperatorscreening/routes.js

```diff
@@ -40,7 +40,7 @@ routes.put('/submissions/:submissionId', middleware.checkRole(['editor']), async
 });
 
 routes.get('/submissions/:submissionId', async (req, res, next) => {
-  await controller.readSubmission(req, res, next);
+  await controller.readSubmissionPublic(req, res, next);
```

Comment:
We don't have anywhere in the app that needs to get everything from /submissions/1234 do we? So all calls to this just returing the "public" view are ok?
I think so, I think the inspection panel and notes panel fetch on their own from the sub routes.

Reply:
Yeah, what I checked in the FE does not use statusCodes/notes/etc. 
See transformToState in minesOperatorScreeningForm.
`  const primary = copy.contacts ? copy.contacts.find(({ contactType }) => contactType === 'PRIMARY') : {};
  const covid = copy.contacts ? copy.contacts.find(({ contactType }) => contactType === 'COVID_COORDINATOR') : {};
  copy.location.startDate = moment(copy.location.startDate).format('YYYY-MM-DD');
  copy.location.endDate = moment(copy.location.endDate).format('YYYY-MM-DD');
  return {
    business: copy.business,
    primaryContact: primary,
    covidContact: covid,
    attestation: copy.attestation,
    location: copy.location
  };
`